resources/views/vendor/backpack/base/inc/sidebar_content.blade.php

```diff
@@ -1,5 +1,63 @@
 <!-- This file is used to store sidebar items, starting with Backpack\Base 0.9.0 -->
-<li class="nav-item"><a class="nav-link" href="{{ backpack_url('dashboard') }}"><i class="la la-home nav-icon"></i> {{ trans('backpack::base.dashboard') }}</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('effect') }}'><i class='nav-icon la la-question'></i> Effects</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('action') }}'><i class='nav-icon la la-question'></i> Actions</a></li>
-<li class='nav-item'><a class='nav-link' href='{{ backpack_url('indicator') }}'><i class='nav-icon la la-question'></i> Indicators</a></li>
\ No newline at end of file
+<!-- <li class="nav-item"><a class="nav-link" href="{{ backpack_url('dashboard') }}"><i class="la la-home nav-icon"></i> {{ trans('backpack::base.dashboard') }}</a></li> -->
+<script src='https://kit.fontawesome.com/a076d05399.js'></script>
```

Comment:
I'd suggest pushing this script onto the 'after_scripts' stack, so this tag will appear at the bottom with the other js tags. This is mainly so the loading of this script doesn't potentially block the rendering of the rest of the page (e.g. in case it can't be found or moves in future).

Although, saying that - I don't know if you can use the @push  directive in an included blade file like this one. If that's not possible, then the way to do it would be to include the script tag in the main template blade file so it renders in the right place, and you still have access to the font-awesome icons here.